### ISSUE TITLE
Whats due

### ISSUE THREAD

**Issue by Caitlynkay** [Media-Ed-Online/intro-web-dev-2018spr-resources#45]
So I just want to make sure I'm not missing anything... the assessments are due tonight, but that's it right? 

I can't find anything else, I just want to make sure i'm not missing something

**Comment by Bourassame** [comment]
Yep! You're right!

**Comment by ewilsey** [comment]
No,  A-3 is also due tonight. Page 4 on Moodle : [Intro to HTML. ](https://moodle.umt.edu/course/view.php?id=19964#section-5)

**Comment by Caitlynkay** [comment]
Oh!!! I didn't know that. Thank you!
